### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by gabays** [katabase/2_CleanedData#4]
Problem while tagging data:
```xml
<desc xml:id="CAT_000428_e122_d1">10 let. <term ana="#document_type_15">aut. sig.</term> au citoyen Rousselin;
 Nantes et La Rochelle, an <date when="1798-1799">VII</date>-an <date when="1798-1799">VII</date>I, 
<measure type="length" unit="p" n="32">32 p.</measure> <measure type="format" unit="f" ana="#document_format_4">in-4</measure>
et in-fol. vignettes.</desc><measure commodity="currency" unit="FRF" quantity="30">30</measure>
```
Look at ` <date when="1798-1799">VII</date>I` and ` let. <term ana="#document_type_15">aut. sig.</term>`

Cf also:
```xml
 2 l. <term ana="#document_type_15">a. s.</term> 
```

For actual case, cf. `CAT_000428_tagged.xml`

**Comment by alexbartz** [comment]
For the term recognition issue, I fixed it with the commit [4ff12ee](https://github.com/katabase/2_CleanedData/commit/4ff12ee7ba6a59066ba4d30b738161eee2c484e1). For a few terms, the script only processed the one with a capital letter (`L. a. s.` and not `l. a. s.`). It's fixed.

**Comment by alexbartz** [comment]
I close the issue since the date error is the same as in https://github.com/katabase/3_TaggedData/issues/2.
